app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -165,7 +165,14 @@ def vm_migrate(vm, options = {})
   end
 
   def unsupported_migration_options
-    [:storage, :respool, :folder, :datacenter, :host_filter]
+    [:storage, :respool, :folder, :datacenter, :host_filter, :cluster]
+  end
+
+  # Migrations are supposed to work only in one cluster. If more VMs are going
+  # to be migrated, all have to live on the same cluster, otherwise they can
+  # not be migrated together.
+  def supports_migrate_for_all?(vms)
```

Comment:
not sure about the name and location of that method

1. it collides with the SupportsFeatureMixin which adds `supports_feature?` methods.
2. this method has nothing to do with the InfraManager, so why is it here?
3. the name does not indicate anyhing about the param. But maybe `migrate` hints at vms?!

@borod108 do you have an idea?